logging/impl.go

```diff
@@ -201,7 +239,7 @@ func (imp *impl) formatw(logLevel Level, traceKey, msg string, keysAndValues ...
 	logEntry.Level = logLevel.AsZap()
 	logEntry.Message = msg
 
-	logEntry.fields = make([]zapcore.Field, 0, len(keysAndValues)/2+1)
+	logEntry.fields = make([]zapcore.Field, 0, len(imp.logFields)+len(keysAndValues)/2+1)
```

Comment:
[super-nit] Is that math still right? Missing some parens?

Reply:
I think this math is correct because `imp.logFields` is a slice of `zapcore.Field` which themselves are key-value pairs, while `keysAndValues` is a `interface{}` where every other element is a key or value.